Sources/Server.swift

```diff
@@ -39,7 +39,8 @@ open class Server {
                 tlsConfig = TLS.Configuration(certificate: cert)
             }
         socket = try Socket.tcpListening(port: port, address: address)
-        queue.async {
+        queue.async { [weak self] in
+            guard let self = self else {return}
```

Comment:
Unfortunately the build failed in Travis. That's probably because of older version of swift does not allow doing `self = self`. However, I think following should work:
```suggestion
            guard let `self` = self else { return }
```